src/util/Util.js

```diff
@@ -231,7 +231,7 @@ class Util extends null {
    */
   static escapeUnderline(text) {
     let i = 0;
-    return text.replace(/__(_)?/g, (_, match) => {
+    return text.replaceAll(/__(_)?/g, (_, match) => {
```

Comment:
Unnecessary, please revert to `replace` as RegExp with global flag will go thru all instances.
```suggestion
    return text.replace(/__(_)?/g, (_, match) => {
```